functions/Resolve-DbaNetworkName.ps1

```diff
@@ -1,90 +1,84 @@
-	
-      .EXAMPLE
-      Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress
+	  .EXAMPLE
+		Resolve-DbaNetworkName -ComputerName ServerA
 
-      Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress
-	
-      .EXAMPLE
-      Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress, sql2014
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for ServerA
 
-      Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress and sql2014
+	  .EXAMPLE
+		Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress
 
-      Get-SqlRegisteredServerName -SqlInstance sql2014 | Resolve-DbaNetworkName
-	
-      Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for all SQL Servers returned by Get-SqlRegisteredServerName
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress
+
+	  .EXAMPLE
+		Resolve-DbaNetworkName -SqlInstance sql2016\sqlexpress, sql2014
+
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for the SQL instance sql2016\sqlexpress and sql2014
+
+		Get-SqlRegisteredServerName -SqlInstance sql2014 | Resolve-DbaNetworkName
+
+		Returns a custom object displaying InputName, ComputerName, IPAddress, DNSHostName, Domain, FQDN for all SQL Servers returned by Get-SqlRegisteredServerName
   #>
 	[CmdletBinding()]
 	param (
 		[parameter(ValueFromPipeline)]
-		[Alias('cn', 'host', 'ServerInstance', 'Server', 'SqlServer')]
-		[DbaInstanceParameter[]]$ComputerName = $env:COMPUTERNAME,
-    	[PSCredential] [System.Management.Automation.CredentialAttribute()]$Credential,
+		[Alias('cn', 'host', 'ServerInstance', 'Server', 'SqlInstance')]
+		[string[]]$ComputerName = $env:COMPUTERNAME,
+		[PSCredential] [System.Management.Automation.CredentialAttribute()]$Credential,
 		[Alias('FastParrot')]
-		[switch]$Turbo
+		[switch]$Turbo,
+		[switch]$Silent
 	)
-	BEGIN
-	{
-		$functionName = (Get-PSCallstack)[0].Command
-	}
-	PROCESS
-	{
-		foreach ($Computer in $ComputerName)
-		{
+
+	process {
+		foreach ($Computer in $ComputerName) {
 			$conn = $ipaddress = $CIMsession = $null
-			
-			if ($Computer.GetType() -eq [Microsoft.SqlServer.Management.Smo.Server])
-			{
+
+			if ($Computer.GetType() -eq [Microsoft.SqlServer.Management.Smo.Server]) {
 				$Computer = $Computer.NetName
 			}
-			
+
 			$OGComputer = $Computer
-			
-			if ($Computer -eq 'localhost' -or $Computer -eq '.')
-			{
+
+			if ($Computer -eq 'localhost' -or $Computer -eq '.') {
 				$Computer = $env:COMPUTERNAME
 			}
-			
+
 			if ($Computer.GetType() -eq [Sqlcollective.Dbatools.Parameter.DbaInstanceParameter]) {
 				$Computer = $Computer.ComputerName
 			}
```

Comment:
Either it will never have this type (right now) or it will always be of that type.
This is residue from a previous hotfix to keep it working and can be removed